cram/cram_io.c

```diff
@@ -1213,6 +1213,7 @@ char *zlib_mem_inflate(char *cdata, size_t csize, size_t *size) {
 }
 #endif
 
+#if !defined(HAVE_LIBDEFLATE) || !(LIBDEFLATE_VERSION_MAJOR > 1 || LIBDEFLATE_VERSION_MINOR > 8)
```

Comment:
Strictly, shouldn't the last bit of this be `( ... || (LIBDEFLATE_VERSION_MAJOR == 1 && LIBDEFLATE_VERSION_MINOR > 8))`?  Although it may work as-is, as I don't think there was a 0.9 release.  Better to be safe, though.

Reply:
Fair comment, although wrong correction I think.  I'll fix.